composer.json

```diff
@@ -22,6 +22,13 @@
         "symfony/phpunit-bridge": "~2.7|~3.0"
     },
     "autoload": {
-        "psr-0": { "Mremi\\UrlShortener": "src" }
+        "psr-4": {
+            "Mremi\\UrlShortener\\": "src/Mremi/UrlShortener"
```

Comment:
You're right, so we should move all files from `src/Mremi/UrlShortener/*` into `src/*` (and same for tests).